zc_events/client.py

```diff
@@ -28,6 +28,21 @@
 
 logger = logging.getLogger('django')
 
+_DEPRECATE_MESSAGE = "DEPRECATION WARNING: Use one of the HTTP verb methods instead"
+
+
+def _deprecated():
+    logger.warning(_DEPRECATE_MESSAGE)
+
+
+def _format_data_structure(data, headers, response_key=False):
```

Comment:
✔️ We talked about this offline. I chose to do it this way because I prefer pure functions of class methods, as it makes it easier for maintenance and we aren't exposing this as a function to be used outside of this file.